src/components/frame/Extensions/layouts/ColonyLayout.tsx

```diff
@@ -5,13 +5,15 @@ import React, {
   type PropsWithChildren,
   useCallback,
   useEffect,
+  useState,
   // useState,
```

Comment:
This can go

Reply:
Removed the comment and rebased, maybe you could re-approve?